src/tactic/omega/eq_elim.lean

```diff
@@ -0,0 +1,353 @@
+/-
+Copyright (c) 2019 Seul Baek. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Seul Baek
+
+A tactic for discharging Presburger arithmetic goals using the Omega test.
+-/
+
+import tactic.omega.clause 
+
+def sgm (v : nat → int) (b : int) (as : list int) (n) := 
```

Comment:
What's the type of `n` and what does `sgm` stand for?

Reply:
I think you're looking at the older commit - in the new one `n` is annotated as `nat`.

Reply:
`sgm` is something I don't have a good name for - the only reason I named it as such is that the text which I got the equality elimination algorithm from (http://www.decision-procedures.org/) called it σ.

Reply:
I'd like a comment pointing this out

Reply:
Added explanation comment